packages/r/rendergraph/xmake.lua

```diff
@@ -0,0 +1,52 @@
+package("rendergraph")
+
+    set_homepage("https://github.com/DragonJoker/RenderGraph/")
+    set_description("Vulkan render graph management library. .")
+
+    set_urls("https://github.com/DragonJoker/RenderGraph.git")
```

Comment:
Add a tar url if it exists

```lua
add_urls("https://github.com/DragonJoker/RenderGraph/archive/refs/tags/$(version).tar.gz",
                "https://github.com/DragonJoker/RenderGraph.git")
add_versions("v1.0.0", "<tar.gz sha256>")
```

Reply:
Well, the commit is not on the tag

Reply:
you can a fake version.
```lua
add_urls("https://github.com/DragonJoker/RenderGraph/archive/refs/tags/$(version).tar.gz",
         "https://github.com/DragonJoker/RenderGraph.git")
add_versions("v1.0.0", "<tar.gz sha256>")
add_versions("v1.0.0-fix", "git commit")
```

Reply:
Given the change from the tag, I've picked it instead of the commit ID.